gradle.properties

```diff
@@ -52,7 +52,7 @@ apacheDirectoryServerVersion=1.5.7
 commonsLangVersion=2.6
 
 # gRPC
-protobufGradlePluginVersion=0.8.12
+protobufGradlePluginVersion=0.8.13
```

Comment:
the build is expected to fail until https://github.com/google/protobuf-gradle-plugin/pull/423 is released